src/Login/Login.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import "./Login.css";
+import {toast, ToastContainer} from "react-toastify";
+import LoginForm from "./LoginForm";
+import "../common/Notify.css"
+
+function Login() {
+
+    const url = window.location.href;
+    const urlParts = url.split('?');
+    const isSuccessSignUp : boolean = urlParts[1] === "successfulSignUp";
+    const isSuccessDelete : boolean = urlParts[1] === "successfulDelete";
+
+    const notifySignUp = () => toast.success('Se ha registrado exitosamente!', {
+        position: "top-center",
+        autoClose: 7000,
+        hideProgressBar: true,
+        closeOnClick: true,
+        pauseOnHover: true,
+        draggable: true,
+        progress: undefined});
+
+    const notifyDelete = () => toast.info('Lamentamos que te hayas ido… Eperamos verte pronto nuevamente!', {
+        position: "top-center",
+        autoClose: 7000,
+        hideProgressBar: true,
+        closeOnClick: true,
+        pauseOnHover: true,
+        draggable: true,
+        progress: undefined});
+
+    if(isSuccessSignUp){
+        notifySignUp();
+        window.history.replaceState("","",urlParts[0])
+    }
+
+    if(isSuccessDelete){
+        notifyDelete();
+        window.history.replaceState("","",urlParts[0])
+    }
+
+    return (
+        <div>
+            <ToastContainer position="top-center" autoClose={5000} hideProgressBar={false} newestOnTop={false}
+                            closeOnClick rtl={false} pauseOnFocusLoss draggable pauseOnHover
+            />
+            <img src={"img/library-1.png"}
+                 srcSet={"img/library-1@2x.png 2x,img/library-1@3x.png 3x"}
+                 className="library-1" alt=""/>
```

Comment:
This background image should be set on the .css rather than here. 

Following the previous comment, if you assign the className "login-screen" to your main div, you can select that div and add a background. 

The css should be something like this:
```
.login-screen {
    height: 100vh;
    width: 100vw;
    background-image: url(../assets/images/your-image.jpg);
}
```
Note 1: Be aware that if you place your images inside the `/public` folder, then URL might be not accessible throw the `url()`, you might want to create an `/assets` folder inside `/src` and there create a `/images` folder to place all your images. 

Note 2: The height and width added on the css are purely for demostration, I notice that the main components (html, main, root, etc) has no height or width set. So is probably better to set these height and width first. (As another PR has some styles changes, for this branch you might want to let the height and width on the login screen and change it latter when the styles fix PR is merged).